### DISCUSSION TITLE
OSC for FBT tracking point possible?

### DISCUSSION THREAD

**Discussion by ballfn** [vrchat-community/osc#47] [Ideas]
As OSC right now can be used for controlling the game directly, it would be cool if we can also extend this to emulate a vive tracker via OSC, this could potentially open up FBT for oculus quest using something like slimeVR

**Comment by Janni9009** [comment]
I'd personally love to see VMC (VirtualMotionCapture's open protocol [Repo](https://github.com/sh-akira/VirtualMotionCaptureProtocol) [Site](https://protocol.vmc.info/)) implemented for in- and output. It's a well established extension of OSC that's commonly used by VTubers specifically for MoCap. This would allow for entirely custom IK, FBT while in desktop, finger tracking, face tracking (although mostly meant for VRoid, so you'd have to translate that with some params), output to a face-cam environment without sacrificing the stream camera, and much more all in a nice and stackable configuration.

**Comment by NotAKidoS** [comment]
![image](https://user-images.githubusercontent.com/37721153/154885074-013a3763-c0fb-4ca7-81f3-8327b99053a1.png)
It will be doable in the future. I've already modified my own version of QuestTrackingSenderForVMT to send OSC directly to VRChat in preparation just to say I used a Quest for Quest FBT.
https://github.com/Unimakura/QuestTrackingSenderForVMT

https://user-images.githubusercontent.com/37721153/154885281-a8170986-7e70-4378-b06a-fa64904ad95f.mp4

**Comment by mlajtos** [comment]
Would it be possible to pair phone and standalone Quest 2 via OSC to do FBT? Like this: https://www.youtube.com/watch?v=EiA2mr3LhEY
